agrc-newsletter/index.html

```diff
@@ -1,78 +1,324 @@
-        <p><a href="https://us2.campaign-archive2.com/?u=0f7773ca3c6d02d1c38851ee9&id=8f7acd3dc9">AGRC Newsletter October 2014</a></p>
-    </div>
-    <div class="grid__col grid__col--1-of-3">
-        <p><a href="https://us2.campaign-archive2.com/?u=0f7773ca3c6d02d1c38851ee9&id=5f6e3d7525">AGRC Newsletter September 2014</a></p>
-        <p><a href="https://us2.campaign-archive1.com/?u=0f7773ca3c6d02d1c38851ee9&id=44d2b9738d">AGRC Newsletter July/August 2014</a></p>
-        <p><a href="https://us2.campaign-archive1.com/?u=0f7773ca3c6d02d1c38851ee9&id=3605bfcc56">AGRC Newsletter June 2014</a></p>
-        <p><a href="https://us2.campaign-archive1.com/?u=0f7773ca3c6d02d1c38851ee9&id=0b7c3450b2">AGRC Newsletter May 2014</a></p>
-        <p><a href="https://us2.campaign-archive2.com/?u=0f7773ca3c6d02d1c38851ee9&id=92e24e5726">AGRC Newsletter April 2014</a></p>
-        <p><a href="https://us2.campaign-archive2.com/?u=0f7773ca3c6d02d1c38851ee9&id=2c2cb0094e">AGRC Newsletter March 2014</a></p>
-        <p><a href="https://us2.campaign-archive2.com/?u=0f7773ca3c6d02d1c38851ee9&id=59d1ad81b7">AGRC Newsletter February 2014</a></p>
-        <p><a href="https://us2.campaign-archive2.com/?u=0f7773ca3c6d02d1c38851ee9&id=0884117836">AGRC Newsletter January 2014</a></p>
-        <p><a href="https://us2.campaign-archive1.com/?u=0f7773ca3c6d02d1c38851ee9&id=8137ffc085">AGRC Newsletter December 2013</a></p>
-        <p><a href="https://us2.campaign-archive1.com/?u=0f7773ca3c6d02d1c38851ee9&id=952ef5e790">AGRC Newsletter November 2013</a></p>
-        <p><a href="https://us2.campaign-archive2.com/?u=0f7773ca3c6d02d1c38851ee9&id=013bd8e649">AGRC Newsletter October 2013</a></p>
-        <p><a href="https://us2.campaign-archive1.com/?u=0f7773ca3c6d02d1c38851ee9&id=0576d3b0d0">AGRC Newsletter August/September 2013</a></p>
-        <p><a href="https://us2.campaign-archive1.com/?u=0f7773ca3c6d02d1c38851ee9&id=7e8f68897c">AGRC Newsletter July 2013</a></p>
-        <p><a href="{% link downloads/AGRCNewsletter_June2013.pdf %}">AGRC Newsletter June 2013</a></p>
-        <p><a href="{% link downloads/AGRCNewsletter_April2013.pdf %}">AGRC Newsletter April 2013</a></p>
-        <p><a href="{% link downloads/AGRCNewsletter_March2013.pdf %}">AGRC Newsletter March 2013</a></p>
-        <p><a href="{% link downloads/AGRCNewsletter_December2012.pdf %}">AGRC Newsletter December 2012</a></p>
-        <p><a href="{% link downloads/AGRCNewsletter_November2012.pdf %}">AGRC Newsletter November 2012</a></p>
-    </div>
+  <div class="grid__col grid__col--1-of-3">
+    <p>
+      <a href="https://mailchi.mp/7563f5b7e011/utah-agrc-latest-updates"
+        >UGRC Newsletter April 2021</a
+      >
+    </p>
+    <p>
+      <a
+        href="https://mailchi.mp/5dae64b97c9b/latest-updates-from-the-utah-agrc"
+        >UGRC Newsletter December 2020</a
+      >
+    </p>
+    <p>
+      <a
```

Comment:
Was this reformatting intentional?

Reply:
IIRC this file was using something like six spaces for tabs. So I ran my formatter. I agree that it looks a little funny, but it's syntactically fine and I felt like it was an improvement.